### PR TITLE
Portfolio Contact form styling and functionality

### PR DESCRIPTION
**Pull request by dselasea**
In this pull request I,
- Added Contact form styling
- Contact form functionality

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -181,17 +182,24 @@ <h4 class="card__title">Skills</h4>
...
                     </div>
                     <div class="form-control">
-                        <textarea name="" id="" cols="30" rows="10" placeholder="Enter text here"></textarea>
+                        <label>
+                            <textarea name="message" cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
+                        </label>
                     </div>
-                    <button class="btn">Get in touch</button>
+                    <button class="btn btn__submit" type="submit">Get in touch</button>
                 </form>
```
Comment (mckent05) [inline index.html:203]: - According to the project requirements, please use the `flex` display property to align the content of your contact in CSS, This allows for more flexibility and responsiveness 😄 😄

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @dselasea,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- The contact form works as expected 👏 👏 
- Descriptive PR title and summary 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -181,17 +182,24 @@ <h4 class="card__title">Skills</h4>
                 <h2 class="heading-2">I'm always interested in hearing about new projects, so if you'd like to chat, please get in touch.</h2>
             </div>
             <div class="contact__form">
-                <form>
+                <form action="https://formspree.io/f/mrgdrewl"
+                method="POST">
                     <div class="form-control">
-                        <input type="text" placeholder="Full name">
+                        <label>
```
Comment (tufoinnkuo10) [inline index.html:188]: - [optional] Great job. the input fields don't need to be rapped in different label tags.the label tag is needed if you want to name your iput field. check [here ](https://www.w3schools.com/html/html_forms_attributes.asp)for more

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @dselasea 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- form submits succesfully
- design well implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**